packages/hasura/package.json

```diff
@@ -38,8 +38,7 @@
   "dependencies": {
     "@golevelup/nestjs-discovery": "workspace:^",
     "@hasura/metadata": "^1.0.2",
-    "js-yaml": "^4.1.0",
-    "zod": "^3.24.2"
```

Comment:
We can always bring it back when there's a potential usage. but for a single `z.enum` it's just an extra dependency to maintain